src/Core/src/Platform/Windows/MauiCombobox.cs

```diff
@@ -0,0 +1,26 @@
+﻿#nullable enable
+using Microsoft.UI.Xaml.Controls;
+
+namespace Microsoft.Maui
+{
+	public class MauiComboBox : ComboBox
```

Comment:
File name should also be `MauiComboBox.cs`

Reply:
good catch